src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/businessprocess/StartBusinessProcessCallbackHandler.java

```diff
@@ -38,14 +40,19 @@ public List<CaseEvent> handledEvents() {
     private final CaseDetailsConverter caseDetailsConverter;
 
     private CallbackResponse startBusinessProcess(CallbackParams callbackParams) {
+        log.info("Starting callback for start business process. Time started {}:", System.nanoTime());
```

Comment:
log statements are usually prefixed with the timestamp. you may want to just calculate the time taken for the method execution. i.e. 

```suggestion
long start = System.nanoTime();
// ...
long finish = System.nanoTime();
long timeElapsed = finish - start;
log.info("Tim taken for start business process {}:", timeElapsed);
```

Reply:
I want to identify where specifically during execution there is a delay. We can tell from Camunda that the business process task is taking 7-15 minutes to complete as Camunda logs tell us:
- When its emitted or started the external task
- When it has finished the external task

In theory, I could remove the timestamps from all logs, and just have a statement - as you say the log will be timestamped on azure logs

Reply:
I could in theory create a stopwatch and log various times when its processing? Open to suggestions

Reply:
> I want to identify where specifically during execution there is a delay. We can tell from Camunda that the business process task is taking 7-15 minutes to complete as Camunda logs tell us:
> 
> * When its emitted or started the external task
> * When it has finished the external task
> 
> In theory, I could remove the timestamps from all logs, and just have a statement - as you say the log will be timestamped on azure logs

This callback is simply adding an entry in Map and returning response back to CCD. This should be pretty fast, can't imagine it taking more than few milliseconds. 

You cam simply add log message here, if that helps debugging, and timestamp will be prefixed automatically.